x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/helpers.test.tsx

```diff
@@ -35,7 +35,7 @@ import {
 import { getThreatMock } from '../../../../../common/detection_engine/schemas/types/threat.mock';
 
 describe('rule helpers', () => {
-  // @ts-expect-error
+  // @ts-expect-error 4.3.5 upgrade - likely requires moment upgrade
```

Comment:
@FrankHassanabad / @yctercero can your team circle back on this?

Reply:
MomentJS is probably never going to expose this and they are winding down support and suggesting people move to an entirely different system altogether. 

https://momentjs.com/docs/#/-project-status/

Reply:
@tylersmalley if it's not a pain, could you add link to this ticket for us to track?

https://github.com/elastic/kibana/issues/120236

Reply:
No problem, added in cb0f129e871b4bb1b7c8298d9739e35f0256292b